src/test/kotlin/cacophony/utils/GraphUtilsTest.kt

```diff
@@ -0,0 +1,70 @@
+package cacophony.utils
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+class GraphUtilsTest {
```

Comment:
please add tests for `getReachableFrom` and `getStronglyConnectedComponents`

i know you are testing `getReachableFrom` in tests for closures but it is a public method so it should have its own test